tasks/dogstatsd.py

```diff
@@ -21,7 +21,7 @@
 # constants
 DOGSTATSD_BIN_PATH = os.path.join(".", "bin", "dogstatsd")
 STATIC_BIN_PATH = os.path.join(".", "bin", "static")
-MAX_BINARY_SIZE = 20 * 1024
+MAX_BINARY_SIZE = 15 * 1024
```

Comment:
The max binary size has to stay > 20; we had build failures otherwise (we have a card to try to figure
out why it grew due to go 1.10)